update-bot/wlmbots/checker_bot.py

```diff
@@ -116,43 +96,59 @@ def generate_config_table(checker_config):
     return text
 
 
+class ResultCollector(object):
```

Comment:
Yeah hmmm I already thought about that. In general it's better to have tiny "executables" and most of the "code logic" encapsulated in classes or modules.